packages/amplify-e2e-tests/src/__tests__/geo-import.test.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  createNewProjectDir,
+  deleteProject,
+  deleteProjectDir,
+  initJSProjectWithProfile,
+  addAuthWithDefault,
+  updateAuthAddUserGroups,
+  amplifyPushWithoutCodegen,
+  addGeofenceCollectionWithDefault,
+  importGeofencesWithDefault,
+  getProjectMeta,
+  getGeofenceCollection,
+  getGeofence,
+  getGeoJSONObj,
+  listGeofences
+} from 'amplify-e2e-core';
+import { existsSync } from 'fs';
+import path from 'path';
+import _ from 'lodash';
+
+describe('amplify geo import', () => {
+  let projRoot: string;
+  let collectionName: string;
+  let region: string;
+  beforeEach(async () => {
+    projRoot = await createNewProjectDir('geo-import-test');
+    await initJSProjectWithProfile(projRoot, {});
+    await addAuthWithDefault(projRoot);
+    await updateAuthAddUserGroups(projRoot, ['admin']);
+    await addGeofenceCollectionWithDefault(projRoot, 'admin');
```

Comment:
These lines need to change as shown in https://github.com/aws-amplify/amplify-cli/pull/9999/commits/1cb54e5839e2187630cda73f11c7073d3f3de4a4